### PR TITLE
Foreign warehouse model was created

### PR DESCRIPTION
**Pull request by ShahanAliyev**

### CODE REVIEW

File: core/admin.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 from core.models import (
     PhonePrefix, WareHouse, Currency, Country,
     ContactUs, ProductType, News, FAQ,
-    CategoryFAQ, Discount, Tariff,
+    CategoryFAQ, Discount, Tariff,ForeignWarehouse
```
Comment (mmzeynalli) [inline core/admin.py:5]: Space after colon. Use autoformatter for smiplicity (black and isort)

File: core/models.py
Comment (mmzeynalli) [inline core/models.py:1]: Write API for this too. I am going to update task

**Review by mmzeynalli — CHANGES_REQUESTED** [review]

File: core/models.py
Comment (mmzeynalli) [reply core/models.py:1]: Also write unit test, and check if user sends request, he indeed gets correct address

File: core/admin.py (hunk 1 above)
Comment (ShahanAliyev) [reply core/admin.py:5]: I remember promised about that. I just want to finish the remaining tasks before them

**Comment by mmzeynalli** [comment]
@ShahanAliyev should I review this task?

File: core/api/urls.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+from core.api.views import get_warehouse, get_warehouses
+
+urlpatterns = [
+    path('warehouses', get_warehouses, name="warehouses" ),
```
Comment (mmzeynalli) [inline core/api/urls.py]: Preferrably, always end the endpoint with /

File: core/api/views.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+from core.api.serializers import GetForeignWarehouseSerializer
+from rest_framework import status
+
+from core.models import ForeignWarehouse
+
+
+@api_view(['GET'])
```
Comment (mmzeynalli) [inline core/api/views.py:9]: Check class based views, but if you want, you still can use function based views

**Review by mmzeynalli — CHANGES_REQUESTED** [review]

**Review by mmzeynalli — APPROVED** [review]

**Pull request merged**